nvdaHelper/remote/sysListView32.cpp

```diff
@@ -33,3 +33,23 @@ error_status_t nvdaInProcUtils_sysListView32_getGroupInfo(handle_t bindingHandle
 	*state=group.state;
 	return 0;
 }
+
+error_status_t nvdaInProcUtils_sysListView32_getColumnContent(handle_t bindingHandle, const unsigned long windowHandle, int item, int subItem, BSTR* text) {
+	LVITEM lvItem = {0};
+	lvItem.mask = LVIF_TEXT | LVIF_COLUMNS;
+	lvItem.iItem = item;
+	lvItem.iSubItem = subItem;
+	lvItem.cchTextMax = 512;
+	wchar_t textBuf[512];

```

Comment:
I wonder whether this will be cleaned up correctly after use. Is there a better approach?

Reply:
That buffer is just an array on the stack. That memory will go away once the function returns.